Tasks/Src/RTOS_BPS.c

```diff
@@ -52,6 +52,15 @@ void RTOS_BPS_MutexPost(BPS_OS_MUTEX* mutex, BPS_OS_OPT options) {
 void RTOS_BPS_TaskCreate(void){
 
 }
-void RTOS_BPS_TimeDelay(void){
 
-}
\ No newline at end of file
+/**
+ * @brief: Creates a Time Delay.
+ * @param dly Defines how many ticks to delay for.
+ * @return none
+ */
+void RTOS_BPS_TimeDelay(BPS_OS_TICK dly){
```

Comment:
Is it worth supporting osTimeDelayHMSM?

Reply:
It would be pretty easy, just some math, but timedelay only works on the order of 10ms it looks like

Reply:
There is a function in micrium with hmsm